api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package clutch.chaos.serverexperimentation.v1;
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "api/v1/annotations.proto";
+
+option go_package = "serverexperimentationv1";
+
+message TestSpecification {
+  oneof config {
+    AbortFault abort = 1;
```

Comment:
@dschaller as discussed offline yesterday, eventually we should be able to reorganize models for server experimentation but I decided against it (when it comes to making the change as part of this PR) to limit the amount of changes in my PR.

Reply:
I agree this should be reorganized in another PR.

I think we can probably:

1. Try to get rid of this wrapper message, although I think @wgallagher needed to add it to handle storing the specs in an `Any` field.
2. Rename the `AbortFault` and `LatencyFault` to something like `AbortExperimentSpec` and `LatencyExperimentSpec`.

`oneof` can't be extended in other proto files, so using a `oneof` would make it hard to add new kinds of experiments in lyft/clutch-private, e.g., squeeze experiments.

Reply:
I think that since this is being changed to an any, we can remove the wrapper message now. The oneof was the problem, not the any

Reply:
regarding the naming, I'd probably stick with 'config': AbortFaultConfig, LatencyFaultConfig, etc.

Reply:
@wgallagher I used this as an opportunity to modify the model layer. I kept the top level `TestConfig` object and made `ClusterPair` a property on the root object (as opposed to a nested property on the fault type - abort/latency). Let me know what you think.